generator/index.js

```diff
@@ -142,6 +142,13 @@ module.exports = async (api, options, rootOptions) => {
       fs.ensureFileSync(api.resolve(path.join(genConfig.dirPathPrefix, 'babel.config.js')));
       await applyBabelConfig(api, genConfig, 'babel.config.js');
     });
+
+    api.render(async () => {
```

Comment:
Can I use custom template without using babel?

Reply:
template copy code block is moved out of if babel